### ISSUE TITLE
Windows GIMP and BIMP extremely slow at processing

### ISSUE THREAD

**Issue by matteusbeus** [alessandrofrancesconi/gimp-plugin-bimp#331]
I used BIMP with GIMP on large batches of PNGs a few years back.  I've come to use it with the latest version of GIMP 2.10.32 for Windows and it seems really slow.  Has something been changed?

**Comment by cbsybase** [comment]
I have seen this as well.  Seems to get slower, the more files to be processed.  Is there an increasing/exponential loop happening?

**Comment by oliveskur** [comment]
Yeah, it's really slow. Only 165 PNG images (320×240px) were processed in 2 hours for watermarking with 6 characters.
